python/test/kv/kvTest.py

```diff
@@ -95,6 +95,18 @@ def testCopy(mode):
         kv.destroy()
     return True
 
+# Test multiprocessing by using subprocess
+def testMultiproc(mode):
+    if mode == 'sharemem':
+        pypath = sys.executable
+        p1 = sp.Popen([pypath, "./multiproc/test1.py"])
```

Comment:
What is the purpose of test1? I don't see anyone read its key.

Reply:
Test 1 puts the first key-val in the shared memory. I just want to make sure that getting the value in the middle of the values (like getting 'test2' in `test3.py`) is correct.